templates/plugin/abrt.yml.erb

```diff
@@ -0,0 +1,23 @@
+---
+:enabled: <%= scope.lookupvar("foreman_proxy::plugin::abrt::enabled") %>
+# Log file for the forwarding script.
+:abrt_send_log_file: <%= scope.lookupvar("foreman_proxy::plugin::abrt::abrt_send_log_file") %>
+# Directory where uReports are stored before they are sent
+:spooldir: <%= scope.lookupvar("foreman_proxy::plugin::abrt::spooldir") %>
+# Merge duplicate reports before sending (requires the satyr gem)
+:aggregate_reports: <%= scope.lookupvar("foreman_proxy::plugin::abrt::aggregate_reports") %>
+# Period (in seconds) after which collected reports are forwarded. Meaningful
+# only if smart-proxy-abrt-send is run as a daemon (not from cron).
+:send_period: <%= scope.lookupvar("foreman_proxy::plugin::abrt::send_period") %>
+<% if scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_url") != :undef -%>
+# FAF server instance the reports will be forwarded to (optional)
+:server_url: <%= scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_url") %>
+# Set to true if FAF server uses self-signed certificate
+:server_ssl_noverify: <%= scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_ssl_noverify") %>
+<% if scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_ssl_cert") != :undef and 
+	scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_ssl_key") != :undef -%>
+# Following two options enable client authentication to FAF server
+:server_ssl_cert: <%= scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_ssl_cert") %>
+:server_ssl_key: <%= scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_ssl_key") %>
+<% end %>
```

Comment:
nitpick, it'd be nice to have an `else` section with commented out settings, in case people go to edit the config file